JavaScript/README.md

```diff
@@ -14,11 +15,17 @@
 - [Edit a Webpage](Edit_Webpage)
 - [Extract Code Comments](Extract_Comments)
 - [Follow automatically on Instagram](Follow_Instagram)
+- [Github Good First Issues extractor](Github_Good_First_Issues)
 - [Github Graph Populator](GitHub_Graph_Populator)
+- [Who unfollowed you on Github](Github_Unfollowers)
 - [HTML to Markdown](Html_to_Markdown)
 - [Like Posts automatically on Instagram](Instagram_Like)
 - [ROT13 Decryption](ROT13_Decryption)
 - [RSA Key Pair Generator](RSA_Key_Pair_Generator)
-- [Send Emails](Send_Emails)     
+- [Send Emails](Send_Emails)    
+- [List of those who starred and un-starred a repo](Stargazers) 
```

Comment:
Follow the alphabetical order here as well. (I  know stargazers is the name, but acc to description you have provided it shouldn't be here)